deploy/template.yaml

```diff
@@ -63,7 +62,7 @@ Mappings:
       nodeOldSpaceLimit: "384"
       nodeEnv: "development"
       desiredTaskCount: 1
-      rumAppId: ""
+      assetsCloudfrontArn: ""
```

Comment:
why the parameter name change?

Reply:
I removed this - it was some WIP that I commited

Reply:
The rumAppId is now an import rather than a map